packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -164,6 +168,7 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
     this._estimatedPageHeight = 0;
     this._focusedIndex = -1;
     this._pageCache = {};
+    this.refs = {};
```

Comment:
Unused?

Reply:
Removed. This was also put there to fix linter issues due to stricter linter settings in our build.  But, it is not needed here.